packages/utils/src/did-utils.ts

```diff
@@ -161,7 +161,7 @@ export function getEthereumAddress(verificationMethod: VerificationMethod): stri
  *
  * @beta This API may change without a BREAKING CHANGE notice.
  */
-export function getChainIdForDidEthr(verificationMethod: _NormalizedVerificationMethod): number {
+export function getChainId(verificationMethod: _NormalizedVerificationMethod): number {
   if (verificationMethod.blockchainAccountId?.includes('@eip155')) {
```

Comment:
may be we can also add is NAN as the parseInt result 

if it is nan, then throw error, hence the Eip712 fallback to use chainId 1?

Reply:
good point